### ISSUE TITLE
fix url in the load_confounds template

### ISSUE THREAD

**Issue by lunebellec** [brainhackorg/global2020#134]
The regular expression to grab urls in the "project_url" field assumes that the URL is the last thing in that field.

This behaviour breaks the URL for the "denoising fMRI" project.

I am going to open a PR fixing this particular URL, may be worth considering getting a more robust parsing rule.

**Comment by SamGuay** [comment]
You are right, the regex implemented is the least robust of _all time_ I'd say. It's done the job so far, but if anyway wants to fix it, they are welcome :smile:. 

I usually fix those problems behind the scene, but didn't catch your project. Sorrry
